### DISCUSSION TITLE
Wordpress PHP Extensions

### DISCUSSION THREAD

**Discussion by max-hill-4** [docker-library/wordpress#914] [General]
When installing some wordpress extensions, they fail to work as currently the php-fpm do not include them. This can be solved by manually creating a dockerfile and adding them, but why not simply have a complete/extra docker image? 

[EDIT] 
I understand its mentioned on the docker hub overview, as explained here:
This image does not provide any additional PHP extensions or other libraries, even if they are required by popular plugins There are an infinite number of possible plugins, and they potentially require any extension PHP supports. Including every PHP extension that exists would dramatically increase the image size.

I disagree, as many wordpress extensions require the same / very similar php extensions, as a extra option? why not have -php-fpm-extended?

Seen similar discussion on [Reddit](https://www.reddit.com/r/docker/comments/1ajqf6l/docker_wordpress_missing_php_extensions/)

**Comment by tianon** [comment]
See https://hub.docker.com/_/wordpress#:~:text=Adding%20additional%20libraries%20/%20extensions (https://github.com/docker-library/docs/tree/master/wordpress#adding-additional-libraries--extensions).
